From 299a32979d13bb134d02ca8fad421c75c05f9e69 Mon Sep 17 00:00:00 2001
From: Luca Boccassi <luca.boccassi@gmail.com>
Date: Wed, 10 Dec 2025 17:08:57 +0000
Subject: [PATCH] hwdb: update autosuspend rules

ninja -C build update-hwdb-autosuspend

(cherry picked from commit edfbd7654f0ff406c47aeb5f63d8ab3eab1041cd)

Resolves: RHEL-72702
---
 hwdb.d/60-autosuspend-fingerprint-reader.hwdb | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
index 6f1099318a..0b04282b88 100644
--- a/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
+++ b/hwdb.d/60-autosuspend-fingerprint-reader.hwdb
@@ -84,8 +84,10 @@ usb:v1C7Ap0571*
 # Supported by libfprint driver egismoc
 usb:v1C7Ap0582*
 usb:v1C7Ap0583*
+usb:v1C7Ap0584*
 usb:v1C7Ap0586*
 usb:v1C7Ap0587*
+usb:v1C7Ap0588*
 usb:v1C7Ap05A1*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
@@ -167,6 +169,7 @@ usb:v04F3p0C99*
 usb:v04F3p0C9D*
 usb:v04F3p0C9F*
 usb:v04F3p0CA3*
+usb:v04F3p0CA8*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
 
@@ -178,10 +181,12 @@ usb:v1C7Ap0603*
 # Supported by libfprint driver focaltech_moc
 usb:v2808p9E48*
 usb:v2808pD979*
+usb:v2808pA27A*
 usb:v2808pA959*
 usb:v2808pA99A*
 usb:v2808pA57A*
 usb:v2808pA78A*
+usb:v2808p1579*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
 
@@ -231,6 +236,7 @@ usb:v27C6p659C*
 usb:v27C6p6A94*
 usb:v27C6p6512*
 usb:v27C6p689A*
+usb:v27C6p66A9*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
 
@@ -242,6 +248,7 @@ usb:v298Dp1010*
 # Supported by libfprint driver realtek
 usb:v0BDAp5813*
 usb:v0BDAp5816*
+usb:v2541pFA03*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
 
@@ -268,11 +275,13 @@ usb:v06CBp0126*
 usb:v06CBp0129*
 usb:v06CBp015F*
 usb:v06CBp0168*
+usb:v06CBp0169*
 usb:v06CBp016C*
 usb:v06CBp0173*
 usb:v06CBp0174*
 usb:v06CBp019D*
 usb:v06CBp019F*
+usb:v06CBp01A0*
  ID_AUTOSUSPEND=1
  ID_PERSIST=0
 
@@ -432,7 +441,6 @@ usb:v16D1p1027*
 usb:v1C7Ap0300*
 usb:v1C7Ap0575*
 usb:v1C7Ap0576*
-usb:v1C7Ap0584*
 usb:v1C7Ap0577*
 usb:v1C7Ap057E*
 usb:v2541p0236*
